### DISCUSSION TITLE
Verze 2

### DISCUSSION THREAD

**Discussion by JarkaP** [LIBCAS/hiko#84] [General]
Co chybí
- [ ] aktualizovaná nápověda
- [ ] chybějící české překlady pro carbon copy,  copy, draft, original, photocopy ze sekce Uchování

**Comment by DanielaIwashita** [comment]
Dobrý den, nejdřív poznámky k editoru fulltextů:

Bylo by možné, aby měl textový editor více funkcí? V souvislosti s básnickými dopisy bychom potřebovali například různé typy odsazení textu (například vložené básně). 

V některých dopisech jsou také nakresleny obrázky. Lze je případně do textu vložit? Do tištěných dopisů je vkládáme, tady by to možná nebylo nutné, pokud bude k dispozici i fotky originálu, ale asi bychom potřebovali nějakou konvenci pro označení obrázku umístěného v textu.

**Reply by JarkaP** [reply]
Pro citace / odsazení atd. slouží fialově označené tlačítko, viz
![obrazek](https://user-images.githubusercontent.com/3973865/154963145-44910ab2-6e0c-4102-80ab-e013c84add70.png)

Nad obrázky se ještě musím zamyslet, zatím mi přijde jako dobrá varianta je nahrát mezi obrazové přílohy a v textu na ně jen odkázat.

**Reply by DanielaIwashita** [reply]
Dobrý den, děkuji za odpověď, ale tato funkce bohužel našemu materiálu nevyhovuje. Nepotřebujeme dát odsazený text do uvozovek, potřebujeme ho prostě jen graficky odsadit. Ani tu čáru po straně nepotřebujeme. Lze to prosím nastavit tak, aby prostě jen některé texty byly o tabelátor posunuty víc napravo? Hodila by se nám i funkce zarovnání uprostřed a vpravo.

**Comment by DanielaIwashita** [comment]
Děkuji, vyzkouším to. D. I.

**Comment by DanielaIwashita** [comment]
Dobrý den,

posílám první seznam postřehů k funkcím editoru od kolegyně, která zkoušela zadat jeden dopis Jakuba Demla z roku 1959:

- datace - u Demla a spol. často církevní datace, složitější v případě pohyblivých svátků (je třeba odpočítat na základě daného svátku v konkrétním roce) - nejde tak ani o jednu z uvedených možností; bylo by asi možné to řešit nějakou dohodnutou poznámkou k datu (píšu "datace podle církevního kalendáře")?
- oslovení - je "Jaroslavu Durychovi pozdravení!" oslovení? 
- místo určení apod. - je v pořádku současně zaškrtnout položku "místo určení je odvozené" a "místo určení je nejisté"? systém to umožňuje, ale nevím, jestli to nebylo myšleno jinak (chtělo by to možná víc vysvětlit, v nápovědě o tom nic není)
- incipit, explicit - máme doplňovat nějaké trojtečky?
- MS manifestation - nevíme, o co jde
- Repozitář/Archiv - nevím, kam dát LA PNP a co by se mělo psát pod Repozitář (nějaký internetový link?)
- Přílohy do 500 kB - nešlo by nabídnout automatické zmenšení?
- aplikace pořád hlásí "je možné, že provedené změny nebudou uloženy", i když jsem předtím klikla na "Uložit"
- dopisy bychom potřebovali formátovat, potřebujeme funkci zarovnání (vpravo/vlevo/uprostřed) a osazení

**Reply by DanielaIwashita** [reply]
pardon, odsazení

**Reply by JarkaP** [reply]
Dobrý den,

- datace – u dat je možné zadat poznámku a datum v podobě, v jaké bylo označené v dopise, tak se mi jako dobrá varianta zdá vyplnit církevní dataci do pole 'datum, jak bylo označeno v dopise' a do kolonek den, měsíc, rok vyplnit datum převedené do gregoriánského kalendáře a případně to patřit poznámkou. Kolegové zpracovávající korespondence ale asi budou schopní to posoudit lépe.
- oslovení – to bych nechala na posouzení kolegům editorům
- nemáme žádnou jednotnou formu napříč korespondencemi, tak záleží asi jen na tom, jak se dohodnete s kolegy v rámci té své (ale je to každopádně možné přidat/odebrat hromadně)
- MS manifestation je jedno z polí převzatých z EMLO, víc budou vědět kolegové, kteří s ním pracovali
- Repozitář/Archiv – opět budou vědet víc kolegové editoři, ale žádné z těch polí není povinné, tak to není třeba vyplňovat. [Tady je příklad](https://historicka-korespondence.cz/browse/letter/?type=tgm_letter&id=1000), jak ta pole používají kolegové z Masarykova ústavu
- Přílohy do 500 kB – přidám si to do návrhů, viz #90 
- "je možné, že provedené změny nebudou uloženy" opravím, pokud ukládáte dopis, tak by to tam být nemělo, je to tam proto, aby editoři omylem neuzavřeli okno s rozpracovaným dopisem
- editor ještě upravím

**Comment by DanielaIwashita** [comment]
Dobrý den, 

mnohokrát děkuji za Vaše odpovědi a těším se na všechna vylepšení databáze. 

Srdečně 

Daniela Iwashita 

[ http://www.ucl.cas.cz/ |    ] 
	
Mgr. Daniela Iwashita, Ph.D. 
oddělení ediční a textologické 
vedoucí týmu pro výzkum korespondence Jakuba Demla 
department of scholarly editing 
head of the Jakub Deml’s letters research group 
Ústav pro českou literaturu AV ČR, v.v.i. 
Institute of Czech Literature of the CAS 
Na Florenci 1420/3, 110 00 Praha 1, Czech Republic 
T +420 222 828 209 
E [ ***@***.*** | ***@***.*** ] 
W [ http://www.ucl.cas.cz/ | www.ucl.cas.cz ] 


Od: "Jarka Pachlová" ***@***.***> 
Komu: "JarkaP/hiko" ***@***.***> 
Kopie: "Daniela Iwashita" ***@***.***>, "Comment" ***@***.***> 
Odeslané: Pondělí, 7. Březen 2022 13:50:32 
Předmět: Re: [JarkaP/hiko] Verze 2 (Discussion #84) 





Dobrý den, 

    * datace – u dat je možné zadat poznámku a datum v podobě, v jaké bylo označené v dopise, tak se mi jako dobrá varianta zdá vyplnit církevní dataci do pole 'datum, jak bylo označeno v dopise' a do kolonek den, měsíc, rok vyplnit datum převedené do gregoriánského kalendáře a případně to patřit poznámkou. Kolegové zpracovávající korespondence ale asi budou schopní to posoudit lépe. 
    * oslovení – to bych nechala na posouzení kolegům editorům 
    * nemáme žádnou jednotnou formu napříč korespondencemi, tak záleží asi jen na tom, jak se dohodnete s kolegy v rámci té své (ale je to každopádně možné přidat/odebrat hromadně) 
    * MS manifestation je jedno z polí převzatých z EMLO, víc budou vědět kolegové, kteří s ním pracovali 
    * Repozitář/Archiv – opět budou vědet víc kolegové editoři, ale žádné z těch polí není povinné, tak to není třeba vyplňovat. [ https://historicka-korespondence.cz/browse/letter/?type=tgm_letter&id=1000 | Tady je příklad ] , jak ta pole používají kolegové z Masarykova ústavu 
    * Přílohy do 500 kB – přidám si to do návrhů, viz [ https://github.com/JarkaP/hiko/issues/90 | #90 ] 
    * "je možné, že provedené změny nebudou uloženy" opravím, pokud ukládáte dopis, tak by to tam být nemělo, je to tam proto, aby editoři omylem neuzavřeli okno s rozpracovaným dopisem 
    * editor ještě upravím 


— 
Reply to this email directly, [ https://github.com/JarkaP/hiko/discussions/84#discussioncomment-2308354 | view it on GitHub ] , or [ https://github.com/notifications/unsubscribe-auth/AX37QU2DHCXBBYMBCSZMKH3U6X3RRANCNFSM5GNLEV7A | unsubscribe ] . 
Triage notifications on the go with GitHub Mobile for [ https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675 | iOS ] or [ https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub | Android ] .
